draft-ietf-cbor-7049bis.md

```diff
@@ -61,7 +61,6 @@ normative:
     date: 2008-08
     seriesinfo:
       IEEE Standard: 754-2008
-  RFC2119:
   RFC3629:
```

Comment:
RFC 2119 still referred to in section 1.2

Reply:
At the beginning of the Terminology section, it has ```{::boilerplate bcp14}```, which covers this.